Libraries/TestLogs.psm1

```diff
@@ -152,168 +152,72 @@ function Collect-CustomLogFile {
 	<#
 	.Synopsis
 		Checks if log file is present in VM and downloads it if so.
-    #>
-    param (
-        [string]$LogsDestination,
+	#>
```

Comment:
I suspect this indentation. Please check again.

Reply:
it was changed to tab, which is our convention